application/src/main/java/com/depromeet/ahmatda/template/service/impl/UserTemplateService.java

```diff
@@ -125,6 +125,23 @@ public void templateAddItem(String userToken, TemplateAddItemRequest templateAdd
         itemAdaptor.createItem(item);
     }
 
+    @Override
+    @Transactional
+    public void templateAddItems(String userToken, TemplateAddItemsRequest templateAddItemsRequest) {
+        Long templateId = templateAddItemsRequest.getUserTemplateId();
+        Template template = templateAdaptor.getTemplateById(templateId)
+                .orElseThrow(() -> new TemplateNotExistException(ErrorCode.TEMPLATE_NOT_FOUND));
+
+        authenticateTemplate(userToken, template);
+
+        for(String itemName : templateAddItemsRequest.getItems()) {
+            Item item = Item.UserTemplateAddItem(templateAddItemsRequest.getUserCategoryId(), template,
+                    itemName, false);
+            itemAdaptor.createItem(item);
+        }
```

Comment:
```suggestion
List<Item> items = templateAddItemsRequest.getItems().stream()
                .map(itemName -> Item.UserTemplateAddItem(templateAddItemsRequest.getUserCategoryId(), template, itemName, false))
                .collect(Collectors.toList());

        itemAdaptor.createAllItem(items);
```

디비 여러번 왔다갔다보단 한번에 넣자!